File: Praktikum 1/Python/PRAK110-2210817310007-HafizhPratamaBudiman.py
import math

a=5
t=12
A=a
B=math.sqrt(a*a+t*t)
C=t
kel=A+B+C
L=A*C/2

print('Diketahui :')
print('Alas =',a,'cm')
print('tinggi =',t,'cm')
print('\nJawab :')
print('Sisi A =',A,'cm')
print('Sisi B = ',end='')
print('%.0f' % B,'cm')
print('Sisi C =',C,'cm')
print('Keliling = ',end='')
print('%.0f' % kel,'cm')
print('Luas = ',end='')
print('%.0f' % L,'cm')

File: Praktikum 1/C/PRAK110-2210817310007-HafizhPratamaBudiman.c
#include <stdio.h>
#include <math.h>

int main() {
    float a,t,A,B,C,kel,L;
    a=5;
    t=12;
    A=a;
    B=sqrt(a*a+t*t);
    C=t;
    kel=A+B+C;
    L=A*C/2;

printf("Diketahui :\n");
printf("Alas = %.0f cm\n",a);
printf("Tinggi = %.0f cm\n\n",t);
printf("Jawab :\n");
printf("Sisi A = %.0f cm\n",A);
printf("Sisi B = %.0f cm\n",B);
printf("Sisi C = %.0f cm\n",C);
printf("Keliling = %.0f cm\n",kel);
printf("Luas = %.0f cm\n",L);
}